puppet/zulip/manifests/sharding.pp

```diff
@@ -0,0 +1,45 @@
+class zulip::sharding {
```

Comment:
Should we call this `tornado_sharding.pp`?  It configures other services to shard requests to Tornado.

Reply:
Makes sense, renamed